utils/gyb_syntax_support/AttributeNodes.py

```diff
@@ -292,6 +292,15 @@
                    '''),
              Child('OriginalDeclName', kind='QualifiedDeclName',
                    description='The referenced original declaration name.'),
+             Child('Period', kind='PeriodToken',
+                   description='''
+                   The period separating the computed var or subscript
+                   and accessor name.
```

Comment:
Minor changes for consistent verbiage:
```suggestion
                   The period separating the original declaration name and the
                   accessor name.
```